docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       CLUSTER_IPFSHTTP_NODEMULTIADDRESS: /dns4/ipfs0/tcp/5001
       CLUSTER_CRDT_TRUSTEDPEERS: '*' # Trust all peers in Cluster
       CLUSTER_RESTAPI_HTTPLISTENMULTIADDRESS: /ip4/0.0.0.0/tcp/9094 # Expose API
+      CLUSTER_RESTAPI_BASICAUTHCREDENTIALS: '{user:secret}' # Require basic auth
```

Comment:
@hsanjuan it seems that this does not work presumably because

> An object mapping "username" to "password". It enables Basic Authentication for the API. Should be used with SSL-enabled or libp2p-endpoints.

I can see there is `--force-http` flag that I think addresses this, however I'm not sure how to set it. Do you mind providing some guidance ?

Reply:
hey @Gozala , try:

```
CLUSTER_RESTAPI_BASICAUTHCREDENTIALS="user:secret,user2:secret2"
```

Sorry that this is not well documented.

Reply:
```suggestion
      CLUSTER_RESTAPI_BASICAUTHCREDENTIALS="user:secret,user2:secret2" # Require basic auth
```

Reply:
Thank you that did the trick